walnut/converter/converter.go

```diff
@@ -172,20 +183,31 @@ func GenerateTaskTemplate(extractedTasks []*task.Class, dumpFileName string) (er
 			return fmt.Errorf("creating file failed: %v", err)
 		}
 	}
+	
 	return
 }
 
-func GenerateWorkflowTemplate(input workflow.Role, dumpFileName string) (err error) {
-	dir, _ := os.Getwd()
-	path := filepath.Join(dir, dumpFileName, "workflows")
-	os.MkdirAll(path, os.ModePerm)
+func GenerateWorkflowTemplate(input workflow.Role, outputDir string) (err error) {
+	path := filepath.Join(outputDir, "workflows")
+	path, _ = homedir.Expand(path)
+	_ = os.MkdirAll(path, os.ModePerm)
 
 	yamlDATA, err := workflow.RoleToYAML(input)
 	if err != nil {
 		return fmt.Errorf("error converting role to YAML: %v", err)
 	}
 
-	fileName := filepath.Join(path, input.GetName() + ".yaml")
+	fileName := filepath.Join(path, input.GetName()+".yaml")
+	if _, err := os.Stat(fileName); !os.IsNotExist(err) {
+		prompt := promptui.Select{
+			Label: fmt.Sprintf("%s already exists, overwrite?", fileName),
+			Items: []string{"yes", "no"},
```

Comment:
Same observation concerning `promptui` as above. The text shown to the user should be `"%s already exists, overwrite? [Y/n]"` with the default option shown uppercase. To be checked if `promptui.Prompt` appends the `[Y/n]` part automatically depending on the default value, or it needs to be added.